triplea_maps.yaml

```diff
@@ -2273,3 +2273,12 @@
     Edition.</p>
     <br><p>Key features include combining UK Pacific and ANZAC into one nation, revised
     order of play for nations, victory objectives and 8 rounds of play.</p>
+ - mapName: AAZ
+   mapCategory: EXPERIMENTAL
+   url: https://github.com/triplea-maps/AAZ/archive/refs/heads/master.zip
+   version: 0
+   img: https://github.com/triplea-maps/AAZ/archive/refs/heads/master/previes.zip
```

Comment:
The img link should be to the preview image. You can find the link by navigating to the file and then right clicking the donwload button and 'copy link'.
That would be: https://github.com/triplea-maps/AAZ/blob/master/Preview.png

Though, the preview image should also have a lower case 'p' on it.